scripts/process_data.py

```diff
@@ -0,0 +1,234 @@
+rnaseq_scaled_zeroone_df.to_csv(rna_out_zeroone_file, sep='\t')
+
+
+# ### Mutation
+
+# In[7]:
+
+# Filter mutation types and generate binary matrix
+mutations = {
+    'Frame_Shift_Del',
+    'Frame_Shift_Ins',
+    'In_Frame_Del',
+    'In_Frame_Ins',
+    'Missense_Mutation',
+    'Nonsense_Mutation',
+    'Nonstop_Mutation',
+    'RNA',
+    'Splice_Site',
+    'Translation_Start_Site',
+}
+
+
+# In[8]:
+
+# Process mutation in long format to dataframe format
+mut_pivot = (mutation_df.query("effect in @mutations")
+                        .groupby(['#sample', 'chr',
+                                  'gene'])
+                        .apply(len).reset_index()
+                        .rename(columns={0: 'mutation'}))
+
+mut_pivot = (mut_pivot.pivot_table(index='#sample',
+                                   columns='gene', values='mutation',
+                                   fill_value=0)
+                      .astype(bool).astype(int))
+
+mut_pivot.to_csv(mut_out_file, sep='\t')
+
+
+# ### Copy Number
+
+# In[9]:
+
+copy_df = copy_df.fillna(0)
+copy_df = copy_df.astype(int)
+copy_df = copy_df.T
+copy_df.columns.name = 'gene'
+copy_df.index.name = 'Sample'
+
+
+# In[10]:
+
+copy_loss_df = copy_df.replace(to_replace=[1, 2, -1], value=0)
+copy_loss_df.replace(to_replace=-2, value=1, inplace=True)
+copy_loss_df.to_csv(copy_loss_out_file, sep='\t')
+
+copy_gain_df = copy_df.replace(to_replace=[-1, -2, 1], value=0)
+copy_gain_df.replace(to_replace=2, value=1, inplace=True)
+copy_gain_df.to_csv(copy_gain_out_file, sep='\t')
+
+
+# ### Clinical Data
+
+# In[11]:
+
+clinical_columns_dict = {
+    'gdc_platform': 'platform',
+    'gdc_center.short_name': 'analysis_center',
+    'gdc_cases.submitter_id': 'sample_id',
+    'gdc_cases.demographic.gender': 'gender',
+    'gdc_cases.demographic.race': 'race',
+    'gdc_cases.demographic.ethnicity': 'ethnicity',
+    'gdc_cases.project.primary_site': 'organ',
+    'gdc_cases.project.project_id': 'acronym',
+    'gdc_cases.tissue_source_site.project': 'disease',
+    'gdc_cases.diagnoses.vital_status': 'vital_status',
+    'gdc_cases.samples.sample_type': 'sample_type',
+    'cgc_case_age_at_diagnosis': 'age_at_diagnosis',
+    'cgc_portion_id': 'portion_id',
+    'cgc_slide_percent_tumor_nuclei': 'percent_tumor_nuclei',
+    'cgc_drug_therapy_drug_name': 'drug',
+    'xml_year_of_initial_pathologic_diagnosis': 'year_of_diagnosis',
+    'xml_stage_event_pathologic_stage': 'stage', 
+}
+
+
+# In[12]:
+
+clinical_sub_df = clinical_df.filter(items=clinical_columns_dict.keys())
+clinical_sub_df = clinical_sub_df.rename(columns=clinical_columns_dict)
+clinical_sub_df.index = clinical_sub_df['sample_id']
+clinical_sub_df.drop('sample_id', axis=1, inplace=True)
+clinical_sub_df['acronym'] = clinical_sub_df['acronym'].str[5:]
+clinical_sub_df.to_csv(clinical_processed_out_file, sep='\t')
+
+
+# ### OncoKB Gene-Types
+
+# In[13]:
+
+response = requests.get('http://oncokb.org/api/v1/genes')
```

Comment:
yeah, I should add it. But it is good to keep it as is to preserve how I downloaded the data in the first place